qiskit/converters/circuits_to_qobj.py

```diff
@@ -42,10 +38,12 @@ def circuits_to_qobj(circuits, user_qobj_header=None, run_config=None,
     Returns:
         Qobj: the Qobj to be run on the backends
     """
+
+    warnings.warn('circuits_to_qobj is not used in 0.8. Use qiskit.compiler.assemble',
+                  DeprecationWarning)
```

Comment:
let's make sure all the warning messages point to the correct function.

Reply:
Good point. So many name changes I missed this one